### ISSUE TITLE
[audioFilePlayer] Problem with iOS 12 or 13 in simulator with audioFilePlayer

### ISSUE THREAD

**Issue by csacchetti** [google/flutter.plugins#121]
I have developed an app that uses audioFilePlayer 2.0.1 and works well with iOS 14. But when I want to test it with an emulator on iOS 12 or iOS 13, nothing is heard.
No errors are reported.
I tried an old release programmed totally differently with the old plugin version 1.3.1 but I get the same result.
This behavior seems strange to me because I started this old app with iOS 13 and everything worked.
I don't have devices with iOS 12 or 13 to be able to test the release mode.
Has anyone experienced these issues and can you help me?
Thanks.

**Comment by monkeyswarm** [comment]
I would not trust emulators/simulators to behave the same as real hardware.
If anyone sees an issue on hardware running iOS 12 or 13, please comment! Leaving open for now.

**Comment by csacchetti** [comment]
Finally today I got an iPhone 6s with iOS 12.1 installed. The plugin works correctly. So with the hardware there are no problems. 
The strange thing is that if I remember correctly, I started using the plugin with the simulator which was in version 13 and it worked regularly. Not now. When you launch Play it is as if it were lost and only after about 10 seconds continues with the  following commands and of course the audio is not heard.
Anyway, the good news is that it works with hardware!
